velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -619,6 +619,23 @@ class PlanBuilder {
       const std::vector<std::string>& outputLayout,
       core::JoinType joinType = core::JoinType::kInner);
 
+  /// Add a NestedLoopJoinNode to join two inputs using filter as join
```

Comment:
```
s/Add/Adds/
```

Reply:
34 out of 36 of function comments in PlanBuilder use 'Add', maybe open a new PR to change them all?